napari/layers/tracks/tracks.py

```diff
@@ -546,7 +549,9 @@ def colormap(self, colormap: str):
     def colormaps_dict(self) -> Dict[str, Colormap]:
         return self._colormaps_dict
 
-    @colormaps_dict.setter
+    # Ignored type because mypy doesn't recognise colormaps_dict as a property
+    # TODO: investigate and fix this - not sure why this is the case?
```

Comment:
Optional: am I misremembering or did you find an mypy/typing issue about settable properties? If so, updating this TODO with that information would be great!.

Reply:
I think this is different to the other issues, which were happening because the property and setter weren't next to each other (https://github.com/python/mypy/issues/1465). I still can't work out what's going on here, so propose we leave it with the TODO comment?

Reply:
I also poked at this, but also couldn't work out what's happening. So agreed the TODO is good for now.